docs-kr/network/nodes.html

```diff
@@ -1029,6 +1030,15 @@ <h3>옵션</h3>
             <td>이 속성은 <code> image </ code> 모양에만 적용됩니다. 참이면 색상 개체가 사용됩니다. 배경색이있는 사각형은 그것 뒤에 그려지며, 테두리를 가지게 됩니다. 이는 모든 테두리 옵션들이 고려되고 있음을 의미하고 있습니다.
             </td>
         </tr>
+        <tr parent="shapeProperties" class="hidden">
+            <td class="indent">shapeProperties.useImageCoords</td>
+            <td>Boolean</td>
+            <td><code>false</code></td>
+            <td>This property only applies to the <code>image</code> shape.
+                When true, the image's (0,0) coords are used for drawing.
+                Note that label positioning may break with this option.
```

Comment:
This should be Korean.

Reply:
I will do my best with google translate.